ruby_03-professional_rails_applications/building-a-vps.markdown

```diff
@@ -308,7 +308,15 @@ them which application process to use when serving requests.
 ### Configuring NGINX
 
 First, we're going to need to know where to tell Passenger to look for an installed version of Ruby.
-Fortunately passenger includes a script for finding this information, which we can use like so:
+Fortunately passenger includes a script for finding this information. Prior to setting that up we'll need to know where our installation of Ruby is hiding:
+
+```sh
+which ruby
+```
+
+You should get something like `/home/deploy/.rvm/rubies/ruby-2.2.1/bin/ruby`
```

Comment:
i dont think this is an accurate output example.

when I run it i get something like:

```
deploy@horace-vps-demo:~$  passenger-config --ruby-command
passenger-config was invoked through the following Ruby interpreter:
  Command: /home/deploy/.rvm/gems/ruby-2.2.1/wrappers/ruby
  Version: ruby 2.2.1p85 (2015-02-26 revision 49769) [x86_64-linux]
  To use in Apache: PassengerRuby /home/deploy/.rvm/gems/ruby-2.2.1/wrappers/ruby
  To use in Nginx : passenger_ruby /home/deploy/.rvm/gems/ruby-2.2.1/wrappers/ruby
  To use with Standalone: /home/deploy/.rvm/gems/ruby-2.2.1/wrappers/ruby /usr/bin/passenger start
```